src/views/Header/HeaderStyleInterpolator.js

```diff
@@ -1,13 +1,37 @@
 import { Dimensions, I18nManager } from 'react-native';
 import getSceneIndicesForInterpolationInputRange from '../../utils/getSceneIndicesForInterpolationInputRange';
 
-const crossFadeInterpolation = (first, index, last) => ({
-  inputRange: [first, index - 0.9, index - 0.2, index, last],
-  outputRange: [0, 0, 0.3, 1, 0],
+function hasHeader(scene) {
+  if (!scene) {
+    return true;
+  }
+  const { descriptor } = scene;
+  return descriptor.options.header !== null;
+}
+
+const crossFadeInterpolation = (scenes, first, index, last) => ({
+  inputRange: [
+    first,
+    first + 0.001,
+    index - 0.9,
+    index - 0.2,
```

Comment:
These numbers scare me. Not sure what the alternative is, but it would be nice if the cross fading felt less hardcoded.

Of course these numbers were already here, so this change seems reasonable

Reply:
yah -- i didn't mess with that too hard. The thing that I added were the extra control points of `first + .001` and `last - .001` to the interpolation